soda/core/soda/soda_cloud/soda_cloud.py

```diff
@@ -49,6 +49,10 @@ def build_scan_results(scan) -> dict:
             if check.outcome is not None and check.archetype is not None
         ]
 
+        profiling = [
```

Comment:
Is that really the root object in which the backend expects us to push samples into?

Reply:
Yes, unfortunately for now we have to put sampling inside profiling for backend expectations.

Reply:
Ok that's actually strange as it was discussed that this would be part of `discover tables` originally. However, this is also not ideal and I think we should see with backend if we can get them to plan something that is more dedicated to receive those files. Here's what I think may happen

In this run, profiling info will not be collected, which means that we will submit nothing about profiling. This needs to be confirmed with backend (I would reach out to @dirkgroenen ), but I would suspect that an empty profiling results set may cause previous metrics to be deleted. If this is not the case, then let's go with this approach as it will work and let's work with backend to clean this up later.

If it causes previously pushed profiling metrics to be deleted, we can't ship this just now IMO until some handling is done on the backend.